packages/react-link/src/components/Link/Link.test.tsx

```diff
@@ -78,4 +78,72 @@ describe('Link', () => {
 
     expect(document.activeElement).toEqual(rootRef.current);
   });
+
+  it('cannot be focused when rendered as an anchor and disabled has been passed to the component', () => {
+    const rootRef = React.createRef<HTMLAnchorElement>();
+
+    wrapper = mount(
```

Comment:
please convert these test to use @testing-library/react

Reply:
Made a new issue to address separately here #19940